release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2139,6 +2158,11 @@ $ ./openshift-install wait-for install-complete
 
 * When using the Technology Preview OCI feature with the oc-mirror CLI plug-in, the mirrored catalog embeds all of the Operator bundles, instead of filtering only on those specified in the image set configuration file. (link:https://issues.redhat.com/browse/OCPBUGS-5085[*OCPBUGS-5085*])
 
+* Fully populated load balancers on {rh-openstack} can be stuck in a pending creation status. This issue can cause problems for clusters that are deployed on {rh-openstack}. To resolve the issue, update your {rh-openstack} packages. link:https://bugzilla.redhat.com/show_bug.cgi?id=2042976[*BZ#2042976*]
+
+* Bulk load-balancer member updates on {rh-openstack} can return a 500 code in response to PUT requests. This issue can cause problems for clusters that are deployed on {rh-openstack}. To resolve the issue, update your {rh-openstack} packages. link:https://bugzilla.redhat.com/show_bug.cgi?id=2100135[*BZ#2100135*]
+
+
```

Comment:
There's [a known issue](https://issues.redhat.com/browse/OCPBUGS-5036) where the external cloud provider doesn't pick up the updated credentials after a rotation. A workaround exists that consists in killing the pod for them to consume the new credential. It could be problematic for customers on OpenStack where the external cloud provider has replaced the in-tree one.

Reply:
Is this specific to ShiftStack, or does it impact other platforms, too?

Reply:
All platforms using external cloud provider. I can't find the list of what are these platforms in 4.12.

Reply:
It looks like this is the [list](https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/release-4.12/vendor/github.com/openshift/library-go/pkg/cloudprovider/external.go#L35-L40) of platforms using external cloud provider in 4.12 by default.

Reply:
See https://github.com/openshift/openshift-docs/pull/54280/commits/7d5e3a7cfcc39dc8a17d6d8ca4d39b1374173fa9

Reply:
Aren't we missing OpenStack in the list of affected platforms?

Reply:
Added. :)